blog-service/2023-11-28-observability.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Logs for Monitoring and Troubleshooting for AWS (Observability)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - aws observability
+  - awso
+  - aws monitoring
+  - aws troubleshooting
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are excited to announce several improvements to our AWS Observability solution.  
+
+* **AWS Observability Hub**. A landing page to access all the capabilities of the Sumo Logic AWS Observability solution, including the out-of-the-box dashboards, alerts, and performance benchmark reports, all from a single place.
+* **Log Searches for Troubleshooting**. Quickly troubleshoot issues within your AWS environment with pre-baked log searches. No need to learn the search query syntax. These new searches can be accessed from the new hub page.
+
+To learn more about the AWS Observability solution and how to set it up, check out our documentation [here](/docs/observability/aws/). 
+
+:::note
+To access these new capabilities, you just have to install the AWS Observability solution. If you have one already installed, congratulations! You'll gain access to these new features automatically. 
+:::
```

Comment:
Because these don't render in rss feeds, I usually use just plain text "NOTE: ..."

Reply:
Oh, I see! I'll change it back.